demos/object_detection_demo/jupyter-python/requirements.txt

```diff
@@ -0,0 +1,10 @@
+requests
+jupyterlab
+ipywidgets
+voila
```

Comment:
This doesn't seem to actually be used anywhere?

Reply:
jupyterlab ipywidgets and voila are not used in the notebook, but they are needed to run the notebook (voila only for the dashboard mode version). Requests is for model downloader. With this requirements.txt the user can immediately run the notebook without needing to first look how to install OpenVINO and the Open Model Zoo requirements. If a user already has this installed in their environment nothing will happen.

Reply:
> jupyterlab ipywidgets and voila are not used in the notebook, but they are needed to run the notebook (voila only for the dashboard mode version).

OK.

> Requests is for model downloader.

In that case I would suggest to not refer to it directly, but to include the MD requirements file with `-r`.

Reply:
Ill change that, thanks!